Documentation/MigrateToComposer/CoWorking.rst

```diff
@@ -1,5 +1,4 @@
-.. include:: /Includes.rst.txt
-.. highlight:: bash
+..  include:: /Includes.rst.txt
 
 ==========
 Co-working
```

Comment:
I am wondering if we can change the title as well? Maybe something like just "Version Control". I don't think "co-working" is really appropriate here. The git repo can be used just for deployment.

Reply:
Yup, thought also about it.